queue/python/queue.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+Queue = {'front': None, 'back': None}
+
+
+class Node:
+    def __init__(self, data, next):
+        self.data = data
+        self.next = next
+
+
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpduneabze/queue/python/queue.py
+++ b/tmp/tmpduneabze/queue/python/queue.py
@@ -7,7 +7,6 @@
     def __init__(self, data, next):
         self.data = data
         self.next = next
-
 
     def Enqueue(Queue, element):
         N = Node(element, None)
```